FORMATTING_GUIDE.md

```diff
@@ -0,0 +1,389 @@
+         ```
+```
+
+**Example with images**:
+
+```
+1. To begin, select the rule in the Rule name column. The rule details pane opens.
+    <img src="{{site.url}}{{site.baseurl}}/images/Security/rule-dup2.png" alt="Opening the rule details pane" width="50%">
+
+1. Select the **Duplicate** button in the upper-right corner of the pane. The Duplicate rule window opens in Visual Editor view and all of the fields are automatically populated with the rule's details. Details are also populated in YAML Editor view.
+    <img src="{{site.url}}{{site.baseurl}}/images/Security/dupe-rule.png" alt="Selecting the duplicate button opens the Duplicate rule window" width="50%">
+```
+
+## Tables
+
+Markdown table columns are automatically sized, and there is no need to specify a different number of dashes in the formatting. 
+
+**Example**:
+
+```
+Header 1 | Header 2
+:--- | :---
+Body 1 | Body 2 that is extremely lengthy but there is no need to specify its width
+```
+
+To insert line breaks within tables, use `<br>`:
+
+```
+Header 1 | Header 2
+:--- | :---
+Body 1 | Body paragraph 1 <br> Body paragraph 2 
+```
+
+To use lists within a table, use `<br>` and `-` :
+
+```
+Header 1 | Header 2
+:--- | :---
+Body 1 | List:<br>- One<br>- Two
+```
+
+
+You can also use `&nbsp;` to insert one space, `&ensp;` to insert two spaces, and `&emsp;` to insert four spaces in table cells.
+
+If you need a list with real bullet points, use the bullet point HTML code:
+
+```
+Header 1 | Header 2
+:--- | :---
+Body 1 | List:<br>&ensp;&#x2022; One<br>&ensp;&#x2022; Two
+```
+
+## Images
+
+Place images in the `images` directory of the documentation website. To refer to images, use relative links (see [Internal links](#internal-links) for more information).
+
+Markdown images are responsive by default. To insert a Markdown image, use the `![<alternate text>](link)` syntax:
+
+```
+![OS branding]({{site.url}}{{site.baseurl}}/images/brand.png)
+```
+
+Markdown uses the image’s actual width to render it. It sets the maximum image width to the width of the main body panel.
+
+If you want to specify the image width or another style, use HTML syntax:
+
+```
+<img src="{{site.url}}{{site.baseurl}}/images/brand.png" alt="OS branding" width="700"/>
+```
+
+You can specify width as a hard-coded number of pixels, as in the preceding example, or as a percentage of the parent width:
+
+```
+<img src="{{site.url}}{{site.baseurl}}/images/brand.png" alt="OS branding" width="70%"/>
+```
+
+To stretch the image to fit the width of the main body panel, use width=“100%”.
+
+To take high-resolution screenshots, in Firefox, right-click on the page and choose “Take Screenshot”.
+
+Image borders are automatic; do not manually add a border to an image.
+
+Always **separate an image from the text with a blank line**:
+
+```
+To send a query to OpenSearch, select the query by placing the cursor anywhere in the query text. Then choose the triangle on the top right of the request or press `Ctrl/Cmd+Enter`:
```

Comment:
Is "press" the right word here?

Reply:
It is now that it's in the style guide :)